ingress/controllers/nginx/controller.go

```diff
@@ -206,6 +239,55 @@ func (lbc *loadBalancerController) sync(key string) {
 	})
 }
 
+func (lbc *loadBalancerController) syncIngress(key string) {
```

Comment:
nit: unless you're planning on using this as a general purpose method, suggest renaming it to syncIngressStatus or updateIngressStatus or something.